composer.json

```diff
@@ -24,6 +28,16 @@
         "react/socket": "^1.12"
     },
     "require-dev": {
-        "phpunit/phpunit": "^9.3 || ^5.7 || ^4.8"
+        "phpunit/phpunit": "^9.6 || ^5.7 || ^4.8.36"
+    },
+    "autoload": {
+        "psr-4": {
+            "ConnectionManager\\Extra\\": "src/"
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "ConnectionManager\\Tests\\Extra\\": "tests/"
+        }
     }
```

Comment:
@yadaiio Duplicate keys, see above. Moving to bottom of file makes sense to me otherwise.

Reply:
Oops, I improved this and I was moving the keys to the bottom of the file. Thank you @clue 👍